app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for i in people:
+        for key in i:
+            Person(i["name"], i["age"])
+    for j in people:
+        try:
+            if j["wife"] is not None:
+                Person.people[j["name"]].wife = Person.people[j["wife"]]
+        except KeyError:
+            pass
+        try:
+            if j["husband"] is not None:
+                Person.people[j["name"]].husband = \
+                    Person.people[j["husband"]]
+        except KeyError:
+            pass
+    result = []
+    for i in range(len(people)):
```

Comment:
You can remove this loop and append people to the `result` list when you create them (line 13)